test/abi/abi.inputParser.js

```diff
@@ -1,7 +1,7 @@
-var assert = require('assert');
+var assert    = require('assert');
 var BigNumber = require('bignumber.js');
-var abi = require('../lib/solidity/abi.js');
-var clone = function (object) { return JSON.parse(JSON.stringify(object)); };
+var abi       = require('../../lib/solidity/abi.js');
+var clone     = function (object) { return JSON.parse(JSON.stringify(object)); };
```

Comment:
I would prefer not to format spaces before require. Is it possible to turn this off in your editor?